src/fuse/mod.rs

```diff
@@ -32,11 +32,13 @@ pub use mount::options_validator;
 mod abi;
 mod argument;
 mod channel;
+mod conversion;
 mod ll_request;
 mod mount;
 mod reply;
 mod request;
 mod session;
+pub use conversion::Cast;
```

Comment:
is `pub use` needed?

Reply:
If you don't use pub, memfs.rs cannot use it.

Reply:
pub(crate) use?

Reply:
Done